sql/moz-fx-data-shared-prod/google_ads_derived/daily_campaign_stats_v1/query.sql

```diff
@@ -1,5 +1,6 @@
 SELECT
   date_day AS `date`,
-  * EXCEPT (date_day)
+  * EXCEPT (date_day),
+  @submission_date AS last_updated_date
```

Comment:
Since the ETL is overwriting the entire table every time I don't think this `last_updated_date` column is going to work as intended (the `min_row_count` check will always pass).

I think you can just use the existing `date` column, as we'd be expecting to have gotten data for that date (and checking the `moz-fx-data-shared-prod.google_ads_derived.daily_campaign_stats_v1` table I'm even already seeing data for today, i.e. the day after `@submission_date`).

Reply:
Ahh very good point, will update that - thanks Sean!